src/chart/options/options.h

```diff
@@ -29,14 +29,35 @@ namespace Vizzu::Gen
 class Options
 {
 public:
+	enum class LegendId : std::underlying_type_t<ChannelId> {
+		color,
+		lightness,
+		size
+	};
+
+	static_assert(std::ranges::all_of(Refl::enum_names<LegendId>,
```

Comment:
Why don't we simply set the enumerator values using the corresponding ChannelId values (`color = ChanneId::color`) instead of checking if they match? That way it would work out of box even if the ChannelId enum order changed instead of resulting in a compiler error.

Reply:
That would be no check that these have to be the same name. It can set the values with your recommendation, but this assert checks the names too.

Reply:
After the transformation, this is the code. I think this is unnecessarily complicated.
```cpp
	enum class LegendId : std::underlying_type_t<ChannelId> {
		color = static_cast<std::underlying_type_t<ChannelId>>(
		    ChannelId::color),
		lightness = static_cast<std::underlying_type_t<ChannelId>>(
		    ChannelId::lightness),
		size = static_cast<std::underlying_type_t<ChannelId>>(
		    ChannelId::size)
	};
```

Reply:
can we have both?

Reply:
it has the benefit of working instead of failing on channelid change. 
but I agree, this is much less readable than it could be without the strict linter config we use. This is a tradeof which has been made already all over the code, not just here.

Reply:
https://github.com/vizzuhq/vizzu-lib/pull/399/commits/4c387d7f169a9fd186c31ac1d3ed59b22493aabe